db/schema/games/0001_schema.up.sql

```diff
@@ -92,6 +92,7 @@ CREATE TABLE IF NOT EXISTS rules (
   terraformCost TEXT,
   starbaseComponentCostFactor REAL,
   packetDecayRate TEXT,
```

Comment:
we aren't storing rules in the database yet. We'll need to blow away this rules table when we eventually do. Can you revert this change?

(also, the way the schema stuff works is you create new files, because the old ones have already been applied. You never modify the existing schema files)

Reply:
Do I also remove the stuff in `wasm\wasm\generated.go`?

Reply:
nah, that's needed. You should run the `go generate ./...` command if you change any models.